src/controller/python/chip/interaction_model/Delegate.h

```diff
@@ -42,18 +42,35 @@ static_assert(std::is_same<chip::EndpointId, uint16_t>::value && std::is_same<ch
               "Members in CommandStatus does not match interaction_model/delegate.py");
 static_assert(sizeof(CommandStatus) == 4 + 2 + 2 + 4 + 4 + 1, "Size of CommandStatus might contain padding");
 
+struct __attribute__((packed)) AttributePath
```

Comment:
supernit: if you put mEndpointId at the end, this may not need padding in general

Reply:
I would suggest keep the current order -- the path is `endpoint/cluster/attribute`